### ISSUE TITLE
Customer error when launching tn-3270 emulator.  Screen shot in the issue.  

### ISSUE THREAD

**Issue by Joe-Winchester** [zowe/zlux#817]
![image](https://user-images.githubusercontent.com/28100302/163399203-b9634002-947d-4770-aeea-df8db789998a.png)

**Comment by Martin-Zeithaml** [comment]
Is it possible to include the dev console screenshot as well?
I had this problem with the terminal because of wrong encoding of minified web application and the error was visible in the console:
![image](https://user-images.githubusercontent.com/66114686/163401065-10c8f8b7-9dea-4bc9-bb0c-8bb03b268d68.png)

**Comment by 1000TurquoisePogs** [comment]
Yes, you must look at the console to find out what the real error is, but likely somehow encoding.
If the terminal encoding is wrong I'd generally suspect the encoding of various things are wrong and the user should check to see if anything in components/app-server has tagging, and if not, to reinstall zowe.
